hadoop-ozone/dist/src/main/smoketest/s3/awss3.robot

```diff
@@ -25,10 +25,11 @@ Suite Setup         Setup s3 tests
 *** Variables ***
 ${ENDPOINT_URL}       http://s3g:9878
 ${BUCKET}             generated
+${BUCKET1}            generated
```

Comment:
Same for this variable as well, And all other files having BUCKET1 can be changed to something more sensible